### PR TITLE
Suggest some readability updates to the intro paragraphs.

### PR DESCRIPTION
**Pull request by hawkinsw**
***
<a href="https://pr-preview.s3.amazonaws.com/hawkinsw/longtasks/pull/69.html" title="Last updated on Jul 3, 2019, 4:56 AM UTC (116998c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/longtasks/69/e3c855f...hawkinsw:116998c.html" title="Last updated on Jul 3, 2019, 4:56 AM UTC (116998c)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -61,25 +61,25 @@ spec:html; type:dfn; for:/; text:browsing context
...
+As the page is loading and while the user is interacting with the page afterwards, both the application and browser queue various events that are then executed by the browser -- e.g. user agent schedules input events based on user’s activity, the application schedules callbacks for requestAnimationFrame and other callbacks, etc. Once in the queue, the browser dequeues these events one-by-one and executes them.
 
-However, some task can take a long time (multiple frames), and if and when that happens, the UI thread is locked and all other tasks are blocked as well. To the user this is commonly visible as a "locked up" page where the browser is unable to respond to user input; this is a major source of bad user experience on the web today:
+However, some task(s) can take a long time (multiple frames) and if/when that happens, the UI thread may become blocked and block all other tasks as well. To the user, this is commonly visible as a "locked up" page where the browser is unable to respond to user input; this is a major source of bad user experience on the web today:
```
Comment (npm1) [inline index.bs]:
```suggestion
However, some tasks can take a long time (multiple frames) and if/when that happens, the UI thread may become blocked and block all other tasks as well. To the user, this is commonly visible as a "locked up" page where the browser is unable to respond to user input; this is a major source of bad user experience on the web today:
```

File: index.bs [hunk 2]
```diff
@@ -61,25 +61,25 @@ spec:html; type:dfn; for:/; text:browsing context
...
+Some applications (and <a href="https://en.wikipedia.org/wiki/Real_user_monitoring">RUM</a> vendors) are already attempting to identify and track cases where "long tasks" happen. For example, one known pattern is to install a ~short periodic timer and inspect the elapsed time between the successive expirations: if the elapsed time is greater than the timer period, then there is high likelihood that one or more long tasks have delayed execution. This approach mostly works but has several bad performance implications: by polling to detect long tasks, the application prevents quiescence and long idle blocks (see requestIdleCallback); it’s bad for battery life; there is no way to know what is causing the delay (e.g. first party, third party code, etc.).
```
Comment (npm1) [inline index.bs]: Not sure if timer expirations is clearer that timer calls... "more long tasks have delayed execution" of what?

Comment (npm1) [inline index.bs]: There's first party or third party, but not sure there's any other option? Hence would prefer the previous format.

File: index.bs (hunk 1 above)
Comment (hawkinsw) [reply index.bs]: Agreed! Look for a version 2 of this patch.

File: index.bs (hunk 2 above)
Comment (hawkinsw) [reply index.bs]: I am adding "execution of event loop". I hope that is reasonable. That was left unsaid in the original version, too. So, I'm not entirely sure what the original author wanted to say, but "event loop" seems reasonable.

Comment (hawkinsw) [reply index.bs]: Agreed!!

**Comment by hawkinsw** [comment]
New version posted! Let me know what you think :-) Thanks for the feedback!

**Pull request closed without merging**